### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by skoolbus39**
Update framework CSS to conform to [IIM code standards](https://github.com/unl/iim-code-standards/blob/gh-pages/index.html)

### CODE REVIEW

File: wdn/templates_4.1/scripts/js-css/extlatin.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .wdn-extlatin {
-    font-family: 'Mercury SSm Ext Latin A', 'Mercury SSm Ext Latin B', Georgia, Baskerville, 'Baskerville Old Face', 'Hoefler Text', Garamond, 'Times New Roman', serif;
+    font-family: 'Mercury SSm A', 'Mercury SSm B', Georgia, serif;
```
Comment (kabel) [inline wdn/templates_4.1/scripts/js-css/extlatin.less]: Did you mean to remove the "Ext Latin" part; aren't these separate fonts?

File: wdn/templates_4.1/scripts/js-css/smallcaps.less [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .wdn-smallcaps {
-    font-family: 'Mercury SSm Small Caps A', 'Mercury SSm Small Caps B', Georgia, Baskerville, 'Baskerville Old Face', 'Hoefler Text', Garamond, 'Times New Roman', serif;
+    font-family: 'Mercury SSm A', 'Mercury SSm B', Georgia, serif;
```
Comment (kabel) [inline wdn/templates_4.1/scripts/js-css/smallcaps.less]: Did you mean to remove the "Small Caps" part?

File: wdn/templates_4.1/scripts/js-css/extlatin.less (hunk 1 above)
Comment (skoolbus39) [inline wdn/templates_4.1/scripts/js-css/extlatin.less]: I most certainly did not. Great catch.

File: wdn/templates_4.1/scripts/js-css/smallcaps.less (hunk 2 above)
Comment (skoolbus39) [inline wdn/templates_4.1/scripts/js-css/smallcaps.less]: Another great catch. I'll push some updates in just a sec.

**Comment by kabel** [comment]
Just an FYI for you. You don't need all these separate commits (looks like you made one per file). Your commits can/should be a logical unit of work being completed. Grouping the files of a related change helps to show the scope of the change(s) you are making.

**Pull request merged**